app/src/js/functional/types.ts

```diff
@@ -32,14 +42,23 @@ export interface LabelType {
 
 export interface TrackType {
   /** ID of the track */
-  id: number
+  id: IdType
   /** type */
   type: string
   /** labels in this track {item index: label id} */
-  labels: {[key: number]: number}
+  labels: {[key: number]: IdType}
+}
+
+export interface ShapeType {
+  /** ID of the shape */
+  id: IdType
+  /** Label ID of the shape */
+  label: IdType[]
+  /** type string of the shape. Value from common/types.ShapeType */
+  shapeType: string
 }
 
-export interface RectType {
+export interface RectType extends ShapeType {
```

Comment:
Why can these no longer be used in BDD types?

Reply:
All the scalabel shape types have `id`, `shapeType`, and `labels` properties now.